cloudflare/resource_cloudflare_record_test.go

```diff
@@ -174,6 +174,30 @@ func TestAccCloudflareRecord_SRV(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareRecord_CAA(t *testing.T) {
+	t.Parallel()
+	var record cloudflare.DNSRecord
+	domain := os.Getenv("CLOUDFLARE_DOMAIN")
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+	resourceName := fmt.Sprintf("cloudflare_record.foobar")
```

Comment:
instead of using static values, let's prefer dynamic and random resource names to prevent potential collisions.

```suggestion
	rnd := generateRandomResourceName()
	name := fmt.Sprintf("cloudflare_record.%s", rnd)
```

I know most of these other resources don't adhear to this however it's how we're looking to move forward with newer tests.

Reply:
This will also require you to update the rest of the test to pass through these values.

Reply:
Hey @jacobbednarz 

I just made another commit that passes this through - I hope this is what you mean

Paul

Reply:
yep, looks great, thanks!